epyqlib/utils/qt.py

```diff
@@ -631,14 +636,21 @@ def dialog_from_file(parent, title, file_name):
     )
 
 
-class PySortFilterProxyModel(QtCore.QSortFilterProxyModel):
-    def __init__(self, *args, filter_column, **kwargs):
-        super().__init__(*args, **kwargs)
+@attr.s(auto_attribs=True)
+class BaseSortFilterProxyModel(QtCore.QSortFilterProxyModel):
```

Comment:
I had imagined this being a generic `QObject` inheriting thing but I kind of forgot about how we want to inherit from something else...  I guess maybe it would still work and then the other class would just get mixed in?

```python
@attr.s(auto_attribs=True)
class AttrsQObject(QtCore.QObject):
    _parent: QtCore.QObject = None

    def __attrs_post_init__(self):
        super().__init__(self._parent)

@attr.s(auto_attribs=True)
class SortFilterProxyModel(AttrsQObject, QtCore.QSortFilterProxyModel):
```

This is really a big picture question though and I can't say I'm hot on the multiple inheritance I just typed here...  maybe it isn't worth it and a pattern is the right approach.